waschente/src/TerminVerwaltung.cpp

```diff
@@ -309,13 +308,13 @@ void TerminVerwaltung::getMaschinen() {
 	QSqlQuery qry = multiquery("SELECT COUNT(*) FROM waschmaschinen", "Fehler beim Holen der Konfiguration!");
 	while (qry.next()) {
 		anzahl = qry.value(0).toInt();
-		status = new bool[anzahl];

```

Comment:
I don’t really understand why this is doing this for every entry. Is `anzahl` changing? Anyway the effect seems to be the same if you’d do this just once after the loop.